charts/appsignal-kubernetes/values.yaml

```diff
@@ -31,8 +31,8 @@ appsignal:
 # Log level for the application
 logLevel: "info"
 
-# Whether resources should be described as managed by Helm
-managed: true
+# Whether this is a kubectl-oriented deployment (adds namespace, removes Helm labels)
+kubectl: false
```

Comment:
This needs to be set, just to not generate helm labels?

Reply:
That, and to not generate the namespace resource. Helm will automatically use or create a namespace through its `--create-namespace` and `--namespace` labels (see the README)